tools/src/main/java/org/infinispan/tools/store/migrator/file/SoftIndexFileStoreIterator.java

```diff
@@ -85,7 +85,7 @@ public MarshallableEntry next() {
                      byte[] serializedValue = EntryRecord.readValue(handle, header, offset);
 
                      offset += header.totalLength();
-                     if (EntryRecord.readEntryHeader(handle, offset) == null) {
+                     if (EntryRecord.readOldEntryHeader(handle, offset) == null) {
```

Comment:
What if the user is migrating a store from the existing version of Infinispan, i.e. 11.x?

Reply:
no sure If I understand what you mean... migrating for 11.0 to 11.x? 

The filenames changed with this PR and the FileProvider only searches for old file names, i.e. the file names previous to this PR.
After this PR, the users don't need to migrate anything. We have the magic number per entry so we are able to read anything.

Reply:
> no sure If I understand what you mean... migrating for 11.0 to 11.x?

Yes. Currently it's possible to migrate from one store implementation to another, e.g. SIFS -> RocksDB, within the latest Infinispan version.

You can determine the version of the source store via `StoreProperties#getMajorVersion` and call `readOldEntryHeader` if `< 11`

Reply:
ok! 
I wasn't aware of that. I need to update SFS too since it is only parsing the old format.